.github/workflows/build-artifact.yml

```diff
@@ -33,12 +33,24 @@ jobs:
         sha=$(echo ${{ github.sha }} | cut -c 1-8)
         echo "::set-env name=artifact_sha::$sha"
 
+    - name: Check dependencies cache
+      id: cache-dependencies
+      uses: actions/cache@v2
+      with:
+        path: ./plugin/ThirdParty
+        key: ${{ runner.os }}-cactbot-${{ hashFiles('./util/fetch_deps.py', './util/DEPS.py') }}
```

Comment:
`runner.os` will always resolve to `windows`, right? Doesn't feel like we need it.

Reply:
Yes, but I think it remain `runner.os` would not bad.
Maybe in the future, cactbot can be built up on `ubuntu` or other os, then this key should be re-usable?